.github/workflows/publish.yml

```diff
@@ -36,11 +36,26 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: node .__publish__/src/publish/post-workflow-details.js
 
+      # Setting the target repo branch will cause the craft config (.craft.yml) to be taken from the checked out branch
+      # By default, we check out the default branch of the repo.
+      # If you need to maintain diverging craft configs on different branches, add your repo and the merge target branch
+      # (i.e. the branch craft will merge the release branch into) into the if condition below.
+      - name: Set target repo checkout branch
+        # Note: Branches registered here MUST BE protected in the target repo!
+        if: |
+          fromJSON(steps.inputs.outputs.result).repo == 'sentry-migr8' && fromJSON(steps.inputs.outputs.result).merge_target == 'main' ||
```

Comment:
I think this first one wouldn't be needed since this is the default branch?

Reply:
removed